index.php

```diff
@@ -36,40 +60,58 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <!-- <script src="https://cdn.tailwindcss.com"></script> -->
     <title>Abby's Seed Vault</title>
     
 </head>
 <body>
-    <h1>Please add New Seeds!</h1>
-        <div>
+    <h1>Abby's Seed Vault</h1>
+    <br/>
+    <h2 class="pageTitle">Please add New Seeds!</h2>
+        <div class="formContainer">
             <form method = "POST">
                 <label for="name">Name of Plant Seed:</label>
                 <input type="text" id="name" name="name" />
-
-                <label for="family_name">What plant family does it come from?:</label>
-                <input type="text" id="family_name" name="family_name" />
+                <br/>
+
+                <label for="genus">What plant family does it come from? </label>
+                    <select id="genus" name="genus">
+                        <option value= "0">Choose a Plant Family</option>
+                        <option value= "1">Umbellifer</option>
+                        <option value= "2">Brassica</option>
+                        <option value= "3">Nightshade</option>
+                        <option value= "4">Curcibits</option>
+                        <option value= "5">Alliums</option>
```

Comment:
Now that you have this working, you should have a think about how you can generate this dropdown list based on a database query.

Imagine that some time in the future, you're going to need to give users the ability to add new genera (is that the right word?????) to the system. With the options hard coded like this, you would need to keep adding new options manually each time a new genus is added.